src/utils.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import Logger from './logger';
+
+let unnamedEditorCount = 0;
+
+export function normalizeArguments( args ) {
+	const normalized = {
+		editors: {},
+
+		// attach( ..., { options } );
+		options: args[ 1 ] || {}
+	};
+
+	let editorOrEditorPairs = args[ 0 ];
+
+	if ( typeof editorOrEditorPairs === 'string' ) {
```

Comment:
I guess that in that scenario the editor (`args[ 1 ]`) is assigned to `normalized.options`. Maybe it could be fixed in the `if` or when the `normalized.options` are defined?